change/@microsoft-fast-foundation-015b56ec-8d8e-4573-9572-9822996f1245.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "getRootActiveElement",
```

Comment:
Might make the comment more descriptive, at least your PR title.

Reply:
yup